### DISCUSSION TITLE
A place for Newbies

### DISCUSSION THREAD

**Discussion by uwbwsvd** [twinbasic/twinbasic#307] [General]
I'm a long term user of vba with MS-Access. I have little bit of knowledge of OOP with for example C#. But I'm completely new to VB. So I have mix of question or knowledge need VB and TwinBasic. Is there a place around the wild of the internet where I can find a home to wider my knowledge? Perhaps in future there will be others too, who looks for.

**Comment by WaynePhillipsEA** [comment]
I suspect many people coming to twinBASIC will have similar background/experience to you, so thanks for opening this discussion.

For the most part, the **_language_** part of VBA and VB6 are mostly identical, so the twinBASIC language itself should feel very familiar to you.  

There are of course practical differences with how projects are built (EXEs/DLLs vs host projects in VBA).  I'm going to link #308 here, as documentation will be important for this project going forward, especially with old official documentation for VB6 getting harder to come by.  [vbforums.com](https://www.vbforums.com) is an active forum and resource for VB6 that you might want to look at.

Right now, the best advice I can give is to open issues or discussions here on GitHub whenever you have questions, as that will also help us identify the areas to focus on when writing the documentation.
